src/vm/eventpipefile.h

```diff
@@ -170,6 +173,10 @@ class EventPipeFile final : public FastSerializableObject
 
     Volatile<LONG> m_metadataIdCounter;
 
+    // Indicates whether the file has been initialized,
+    // e.g., has the header been sent to the stream?
+    Volatile<BOOL> m_isInitialized;
```

Comment:
Does it need to be `Volatile` (maybe I am missing part of the picture here)?

Reply:
Removed; it isn't actually necessary I don't think